### DISCUSSION TITLE
Bug wth Wire:Submit in Tag <form>

### DISCUSSION THREAD

**Discussion by devajmeireles** [livewire/livewire#4554] [7. Bugs (archived)]
I recently opened this discussion: https://github.com/livewire/livewire/discussions/4552 with the purpose of understanding what was happening with a specific form in my application that received the “readonly” property when it was sent to the backend. After a lot of research I ended up understanding that the problem happens when I use “wire:submit.prevent” as part of the “form” tag of the form instead of using it in the “save” button, for example.

I managed to discover this problem by looking at another form from a live application (repository: https://github.com/themesberg/volt-laravel-dashboard - example problem: https://volt-laravel-admin-dashboard.updivision.com/login) which also uses “wire:submit.prevent” inside the form tag instead of using it in the “login” button. 

**In my opinion the main problems are:**

**1.** There is no explanation of this in the documentation – at least I haven't found it. I will send a PR with the explanation of this feature.

**2.** Have the possibility to automatically add OR NOT readonly in forms that have “wire:submit.prevent” in the “form” tag.

**3. The main problem:** Using “wire:submit.prevent” in the “form” tag makes the “wire:loading” inside some form element not work for the first request, only if, for example, there is an error on the form that requires it to be submitted a second time.

**Comment by devajmeireles** [comment]
PR to the docs: https://github.com/livewire/docs/pull/405

C/C @calebporzio @joshhanley

**Comment by joshhanley** [comment]
@ajmeireles in future, can I suggest just updating your original discussion, rather than opening a new one, if you find out more details. You should be able to change the category as well, but if you can't, an admin can change it for you.

I think you are right that Livewire does that to any elements in a form inputs/ buttons, to prevent changes while the request is out.

I'm currently working on a new docs page for form elements, so I will also add a note in there about it.

**Reply by devajmeireles** [reply]
> @ajmeireles in future, can I suggest just updating your original discussion, rather than opening a new one, if you find out more details. You should be able to change the category as well, but if you can't, an admin can change it for you.
> 
> I think you are right that Livewire does that to any elements in a form inputs/ buttons, to prevent changes while the request is out.
> 
> I'm currently working on a new docs page for form elements, so I will also add a note in there about it.

I've sent a PR to contribuite with the livewire docs.
